modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserServiceWhenAddingDefaultGroupTest.java

```diff
@@ -51,7 +52,7 @@ public class UserServiceWhenAddingDefaultGroupTest {
 
 	@Before
 	public void setUp() throws Exception {
-		_company = CompanyTestUtil.addCompany();
+		_company = _companyLocalService.getCompany(TestPropsValues.getCompanyId());
```

Comment:
Is needed to save it as global or now (because we don't have to delete it) we can use a local variable?